packages/aml-checklist/src/components/ui/forms/form-schema/validation-schema.ts

```diff
@@ -0,0 +1,15 @@
+import * as Yup from 'yup'
+import formFields from './form-fields'
+import errorMessages from '@/constants/error-messages'
+import { isValidUploadForm } from '@/utils/validates'
+
+const { documentId } = formFields
+
+/* istanbul ignore next */
+const validationSchema = Yup.object().shape({
+  [documentId.name]: Yup.string()
+    .required(errorMessages.FIELD_REQUIRED)
+    .test('isValidBase64', 'Wrong file type', value => isValidUploadForm(value)),
```

Comment:
Please add a new constant for 'wrong file type' error message to @/constants/error-messages and use from there